app/controllers/generic_object_definition_controller.rb

```diff
@@ -286,4 +288,32 @@ def textual_group_list
   end
 
   helper_method :textual_group_list
+
+  def breadcrumbs_options
+    {
+      :breadcrumbs => [
+        {:title => _("Automation")},
+        {:title => _("Automate")},
+        {:title => _("Generic Objects")},
+      ],
+      :record_info => (
+        unless @generic_object_definition.nil?
+          {
+            "title" => @generic_object_definition.name,
+            "id"    => @generic_object_definition.id
+          }
+        end
+      ),
+    }
+  end
+
+  def build_breadcrumbs_from_tree(breadcrumbs)
```

Comment:
This method overrides a method from the mixin. In other controllers three hierarchy is stored in `@trees`, in this controller there is`@tree`. Otherwise, this method builds tree hierarchy and call current_tree_path to generate breadcrumbs from it.